packages/playground/wordpress/src/index.ts

```diff
@@ -131,27 +147,42 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 			if (!$user) {
 				return;
 			}
+			/**
+			 * This approach is described in a comment on
+			 * https://developer.wordpress.org/reference/functions/wp_set_current_user/
+			 */
 			wp_set_current_user( $user->ID, $user->user_login );
 			wp_set_auth_cookie( $user->ID );
 			do_action( 'wp_login', $user->user_login, $user );
 			setcookie('playground_auto_login_already_happened', '1');
-		}
 
+			/**
+			 * Reload page to ensure the user is logged in correctly.
+			 * WordPress uses cookies to determine if the user is logged in,
+			 * so we need to reload the page to ensure the cookies are set.
+			 *
+			 * Both WordPress home url and REQUEST_URI may include the site scope
+			 * subdirectory.
+			 * To prevent the redirect from including two scopes, we need to
+			 * remove the scope from the REQUEST_URI if it's present.
+			 */
+			$redirect_url = $_SERVER['REQUEST_URI'];
+			if (strpos($redirect_url, '/scope:') === 0) {
+				$parts = explode('/', $redirect_url);
+				$redirect_url = '/' . implode('/', array_slice($parts, 2));
+			}
+			wp_redirect(
+				home_url($redirect_url),
```

Comment:
Is the home_url() call necessary here?

Reply:
I think this is about providing an absolute URL to Safari as @bgrgicak mentioned above. Can you add this info as a comment in a follow-up PR?

Reply:
Good point @akirk, I can explain this in a comment.

Reply:
I documented it in this PR https://github.com/WordPress/wordpress-playground/pull/1971/files#diff-8afd615c3847003caebf2b35102f3a2a107ad9647e095495ecc9302862802191R166